### ISSUE TITLE
Full Node crashed on `Seed mode`

### ISSUE THREAD

**Issue by kidinamoto01** [tendermint/tendermint#1543]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
Bug Report

**Tendermint version** (use `tendermint version` or `git rev-parse --verify HEAD` if installed from source):
v0.19.1

**ABCI app** (name for built-in, URL for self-written if it's publicly available):
Gaiad v0.16-0-dev

**Merkleeyes version** (use `git rev-parse --verify HEAD`, skip if you don't use it):

**Environment**:
- **OS** (e.g. from /etc/os-release): ubuntu 14.06
- **Install tools**: dep
- **Others**:

**What happened**:
after setting  `pex = true`&`seed_mode = true`, I started a full node. Then the node crashed after running for several hours. 

**What you expected to happen**:
I can get most node IPs in my addrbook.

**How to reproduce it** (as minimally and precisely as possible):
run a mode in seed mode 
**Logs (you can paste a part showing an error or attach the whole file)**:
```
I[05-08|14:06:26.354] Stopping gossipVotesRoutine for peer         module=consensus peer="Peer{MConn{128.199.121.8:46656} f4c98b895e6cc456814789fcce1b9112d123e0eb out}"
I[05-08|14:06:26.354] Stopping gossipDataRoutine for peer          module=consensus peer="Peer{MConn{207.154.243.140:46656} 45226d5ee2d27c93ec64a674f6bbf6c8f6f598c5 out}"
I[05-08|14:06:26.354] Stopping gossipVotesRoutine for peer         module=consensus peer="Peer{MConn{139.162.19.186:46656} 4a82e59127e198fd27d3409af94079a83cac7bcc out}"
I[05-08|14:06:26.355] Stopping gossipDataRoutine for peer          module=consensus peer="Peer{MConn{138.68.77.24:46656} 1ebc5ca705b3ae1c06a0888ff1287ada82149dc3 out}"
I[05-08|14:06:26.355] Stopping gossipVotesRoutine for peer         module=consensus peer="Peer{MConn{207.154.243.140:46656} 45226d5ee2d27c93ec64a674f6bbf6c8f6f598c5 out}"
I[05-08|14:06:26.356] Stopping gossipVotesRoutine for peer         module=consensus peer="Peer{MConn{138.68.77.24:46656} 1ebc5ca705b3ae1c06a0888ff1287ada82149dc3 out}"
I[05-08|14:06:26.371] Stopping gossipDataRoutine for peer          module=consensus peer="Peer{MConn{45.77.53.208:46656} aef085c4bfed0c1ffc6705f2e1e3bf85e5164600 out}"
I[05-08|14:06:26.372] Stopping gossipVotesRoutine for peer         module=consensus peer="Peer{MConn{34.205.39.35:46656} eae1dcdfa096571001072417147df0b686fdff48 out}"
I[05-08|14:06:26.399] Stopping gossipDataRoutine for peer          module=consensus peer="Peer{MConn{172.105.224.9:46656} 856de8fffc3d9faceefd5e2e38c0e886d507b083 out}"
I[05-08|14:06:26.399] Stopping gossipDataRoutine for peer          module=consensus peer="Peer{MConn{158.69.63.13:46656} ea3163772854232dd7341fef689302750db549f4 out}"
I[05-08|14:06:26.399] Stopping gossipDataRoutine for peer          module=consensus peer="Peer{MConn{35.202.97.136:46656} 678503e6c8f50db7279c7da3cb9b072aac4bc0d5 out}"
I[05-08|14:06:26.399] Stopping gossipDataRoutine for peer          module=consensus peer="Peer{MConn{209.250.227.203:46656} 38dda2d2e814894e2fa901b03c3babdc0de5a669 out}"
I[05-08|14:06:26.399] Stopping gossipDataRoutine for peer          module=consensus peer="Peer{MConn{35.184.154.225:46656} 30b45459e4881680c0ef1750fde136fefa6c3b98 out}"
I[05-08|14:06:26.399] Stopping gossipDataRoutine for peer          module=consensus peer="Peer{MConn{34.235.130.1:46656} 79466a03e9d4b4648a7dd8cead1fa7121ce76ee3 out}"
I[05-08|14:06:26.399] Stopping gossipVotesRoutine for peer         module=consensus peer="Peer{MConn{172.105.224.9:46656} 856de8fffc3d9faceefd5e2e38c0e886d507b083 out}"
I[05-08|14:06:26.399] Stopping gossipVotesRoutine for peer         module=consensus peer="Peer{MConn{35.184.154.225:46656} 30b45459e4881680c0ef1750fde136fefa6c3b98 out}"
I[05-08|14:06:26.400] Stopping gossipVotesRoutine for peer         module=consensus peer="Peer{MConn{35.202.97.136:46656} 678503e6c8f50db7279c7da3cb9b072aac4bc0d5 out}"
I[05-08|14:06:26.400] Stopping gossipVotesRoutine for peer         module=consensus peer="Peer{MConn{45.77.53.208:46656} aef085c4bfed0c1ffc6705f2e1e3bf85e5164600 out}"
I[05-08|14:06:26.400] Stopping gossipVotesRoutine for peer         module=consensus peer="Peer{MConn{209.250.227.203:46656} 38dda2d2e814894e2fa901b03c3babdc0de5a669 out}"
I[05-08|14:06:26.402] Stopping gossipVotesRoutine for peer         module=consensus peer="Peer{MConn{158.69.63.13:46656} ea3163772854232dd7341fef689302750db549f4 out}"
I[05-08|14:06:26.403] Stopping gossipDataRoutine for peer          module=consensus peer="Peer{MConn{18.208.82.127:46656} 801bdd7d17e01466feeae9e6d4d46caa1d0c9e0c out}"
I[05-08|14:06:26.434] Stopping gossipVotesRoutine for peer         module=consensus peer="Peer{MConn{138.197.161.235:46656} 03b977aaecb87315fb64d4925c3cd72305129f2a out}"
I[05-08|14:06:26.434] Stopping gossipDataRoutine for peer          module=consensus peer="Peer{MConn{31.187.71.49:46656} b70f3ca576591d84dc49f442399c856340be603e out}"
I[05-08|14:06:26.434] Stopping gossipVotesRoutine for peer         module=consensus peer="Peer{MConn{34.235.130.1:46656} 79466a03e9d4b4648a7dd8cead1fa7121ce76ee3 out}"
I[05-08|14:06:26.434] Stopping gossipDataRoutine for peer          module=consensus peer="Peer{MConn{138.197.161.235:46656} 03b977aaecb87315fb64d4925c3cd72305129f2a out}"
I[05-08|14:06:26.434] Stopping gossipVotesRoutine for peer         module=consensus peer="Peer{MConn{31.187.71.49:46656} b70f3ca576591d84dc49f442399c856340be603e out}"
I[05-08|14:06:26.434] Stopping gossipVotesRoutine for peer         module=consensus peer="Peer{MConn{18.208.82.127:46656} 801bdd7d17e01466feeae9e6d4d46caa1d0c9e0c out}"
I[05-08|14:06:26.440] Stopping gossipDataRoutine for peer          module=consensus peer="Peer{MConn{116.62.62.39:46656} aabf05a67b2f399807dc602d05bf97b0ed283ac2 out}"
I[05-08|14:06:26.440] Stopping gossipVotesRoutine for peer         module=consensus peer="Peer{MConn{116.62.62.39:46656} aabf05a67b2f399807dc602d05bf97b0ed283ac2 out}"
I[05-08|14:06:26.537] Stopping queryMaj23Routine for peer          module=consensus peer="Peer{MConn{35.184.154.225:46656} 30b45459e4881680c0ef1750fde136fefa6c3b98 out}"
I[05-08|14:06:26.826] Successful handshake with peer               module=p2p peer=35.202.97.136:46656 peerNodeInfo="NodeInfo{id: 678503e6c8f50db7279c7da3cb9b072aac4bc0d5, moniker: gaia-5000, network: gaia-5000 [listen 10.128.0.9:46656], version: 0.19.1 ([amino_version=0.9.8 p2p_version=0.5.0 consensus_version=v1/0.2.2 rpc_version=0.7.0/3 tx_index=on rpc_addr=tcp://0.0.0.0:46657])}"
I[05-08|14:06:26.826] Starting Peer                                module=p2p peer=35.202.97.136:46656 impl="Peer{MConn{35.202.97.136:46656} 678503e6c8f50db7279c7da3cb9b072aac4bc0d5 out}"
I[05-08|14:06:26.826] Starting MConnection                         module=p2p peer=35.202.97.136:46656 impl=MConn{35.202.97.136:46656}
I[05-08|14:06:26.826] Added peer                                   module=p2p peer="Peer{MConn{35.202.97.136:46656} 678503e6c8f50db7279c7da3cb9b072aac4bc0d5 out}"
I[05-08|14:06:26.826] Dialing peer                                 module=p2p address=856de8fffc3d9faceefd5e2e38c0e886d507b083@172.105.224.9:46656
I[05-08|14:06:26.917] Stopping queryMaj23Routine for peer          module=consensus peer="Peer{MConn{138.68.77.24:46656} 1ebc5ca705b3ae1c06a0888ff1287ada82149dc3 out}"
I[05-08|14:06:26.930] Successful handshake with peer               module=p2p peer=172.105.224.9:46656 peerNodeInfo="NodeInfo{id: 856de8fffc3d9faceefd5e2e38c0e886d507b083, moniker: forbole, network: gaia-5000 [listen 172.105.224.9:46656], version: 0.19.1 ([amino_version=0.9.8 p2p_version=0.5.0 consensus_version=v1/0.2.2 rpc_version=0.7.0/3 tx_index=on rpc_addr=tcp://0.0.0.0:46657])}"
I[05-08|14:06:26.930] Starting Peer                                module=p2p peer=172.105.224.9:46656 impl="Peer{MConn{172.105.224.9:46656} 856de8fffc3d9faceefd5e2e38c0e886d507b083 out}"
I[05-08|14:06:26.930] Starting MConnection                         module=p2p peer=172.105.224.9:46656 impl=MConn{172.105.224.9:46656}
I[05-08|14:06:26.930] Added peer                                   module=p2p peer="Peer{MConn{172.105.224.9:46656} 856de8fffc3d9faceefd5e2e38c0e886d507b083 out}"
I[05-08|14:06:26.930] Dialing peer                                 module=p2p address=45226d5ee2d27c93ec64a674f6bbf6c8f6f598c5@207.154.243.140:46656
I[05-08|14:06:26.937] Stopping queryMaj23Routine for peer          module=consensus peer="Peer{MConn{45.77.53.208:46656} aef085c4bfed0c1ffc6705f2e1e3bf85e5164600 out}"
I[05-08|14:06:27.096] Stopping queryMaj23Routine for peer          module=consensus peer="Peer{MConn{31.187.71.49:46656} b70f3ca576591d84dc49f442399c856340be603e out}"
I[05-08|14:06:27.151] Stopping queryMaj23Routine for peer          module=consensus peer="Peer{MConn{209.250.227.203:46656} 38dda2d2e814894e2fa901b03c3babdc0de5a669 out}"
I[05-08|14:06:27.160] Stopping queryMaj23Routine for peer          module=consensus peer="Peer{MConn{158.69.63.13:46656} ea3163772854232dd7341fef689302750db549f4 out}"
I[05-08|14:06:27.204] Timed out                                    module=consensus dur=969.423761ms height=92671 round=0 step=RoundStepNewHeight
I[05-08|14:06:27.205] enterNewRound(92671/0). Current: 92671/0/RoundStepNewHeight module=consensus
I[05-08|14:06:27.206] enterPropose(92671/0). Current: 92671/0/RoundStepNewRound module=consensus
I[05-08|14:06:27.348] Stopping queryMaj23Routine for peer          module=consensus peer="Peer{MConn{35.202.97.136:46656} 678503e6c8f50db7279c7da3cb9b072aac4bc0d5 out}"
I[05-08|14:06:27.445] Stopping queryMaj23Routine for peer          module=consensus peer="Peer{MConn{172.105.224.9:46656} 856de8fffc3d9faceefd5e2e38c0e886d507b083 out}"
I[05-08|14:06:27.485] Stopping queryMaj23Routine for peer          module=consensus peer="Peer{MConn{128.199.121.8:46656} f4c98b895e6cc456814789fcce1b9112d123e0eb out}"
I[05-08|14:06:27.737] Stopping queryMaj23Routine for peer          module=consensus peer="Peer{MConn{104.237.6.136:46656} 5917f03dc46775b2e649f775969f6b6ae2bf4bf4 out}"
I[05-08|14:06:27.749] Successful handshake with peer               module=p2p peer=207.154.243.140:46656 peerNodeInfo="NodeInfo{id: 45226d5ee2d27c93ec64a674f6bbf6c8f6f598c5, moniker: 🔥 Nylira Validator, network: gaia-5000 [listen 207.154.243.140:46656], version: 0.19.1 ([amino_version=0.9.8 p2p_version=0.5.0 consensus_version=v1/0.2.2 rpc_version=0.7.0/3 tx_index=on rpc_addr=tcp://0.0.0.0:46657])}"
I[05-08|14:06:27.749] Starting Peer                                module=p2p peer=207.154.243.140:46656 impl="Peer{MConn{207.154.243.140:46656} 45226d5ee2d27c93ec64a674f6bbf6c8f6f598c5 out}"
I[05-08|14:06:27.749] Starting MConnection                         module=p2p peer=207.154.243.140:46656 impl=MConn{207.154.243.140:46656}
I[05-08|14:06:27.749] Added peer                                   module=p2p peer="Peer{MConn{207.154.243.140:46656} 45226d5ee2d27c93ec64a674f6bbf6c8f6f598c5 out}"
I[05-08|14:06:27.749] Dialing peer                                 module=p2p address=38dda2d2e814894e2fa901b03c3babdc0de5a669@209.250.227.203:46656
I[05-08|14:06:27.775] Stopping queryMaj23Routine for peer          module=consensus peer="Peer{MConn{138.197.161.235:46656} 03b977aaecb87315fb64d4925c3cd72305129f2a out}"
I[05-08|14:06:28.039] Stopping queryMaj23Routine for peer          module=consensus peer="Peer{MConn{34.235.130.1:46656} 79466a03e9d4b4648a7dd8cead1fa7121ce76ee3 out}"
I[05-08|14:06:28.234] Stopping queryMaj23Routine for peer          module=consensus peer="Peer{MConn{116.62.62.39:46656} aabf05a67b2f399807dc602d05bf97b0ed283ac2 out}"
I[05-08|14:06:28.541] Successful handshake with peer               module=p2p peer=209.250.227.203:46656 peerNodeInfo="NodeInfo{id: 38dda2d2e814894e2fa901b03c3babdc0de5a669, moniker: ion.dokia-capital-1, network: gaia-5000 [listen 209.250.227.203:46656], version: 0.19.1 ([amino_version=0.9.8 p2p_version=0.5.0 consensus_version=v1/0.2.2 rpc_version=0.7.0/3 tx_index=on rpc_addr=tcp://0.0.0.0:46657])}"
I[05-08|14:06:28.541] Starting Peer                                module=p2p peer=209.250.227.203:46656 impl="Peer{MConn{209.250.227.203:46656} 38dda2d2e814894e2fa901b03c3babdc0de5a669 out}"
I[05-08|14:06:28.541] Starting MConnection                         module=p2p peer=209.250.227.203:46656 impl=MConn{209.250.227.203:46656}
I[05-08|14:06:28.541] Added peer                                   module=p2p peer="Peer{MConn{209.250.227.203:46656} 38dda2d2e814894e2fa901b03c3babdc0de5a669 out}"
I[05-08|14:06:28.541] Dialing peer                                 module=p2p address=f4c98b895e6cc456814789fcce1b9112d123e0eb@128.199.121.8:46656
I[05-08|14:06:28.880] Successful handshake with peer               module=p2p peer=128.199.121.8:46656 peerNodeInfo="NodeInfo{id: f4c98b895e6cc456814789fcce1b9112d123e0eb, moniker: EddyG, network: gaia-5000 [listen 128.199.121.8:46656], version: 0.19.1 ([amino_version=0.9.8 p2p_version=0.5.0 consensus_version=v1/0.2.2 rpc_version=0.7.0/3 tx_index=on rpc_addr=tcp://0.0.0.0:46657])}"
I[05-08|14:06:28.880] Starting Peer                                module=p2p peer=128.199.121.8:46656 impl="Peer{MConn{128.199.121.8:46656} f4c98b895e6cc456814789fcce1b9112d123e0eb out}"
I[05-08|14:06:28.880] Starting MConnection                         module=p2p peer=128.199.121.8:46656 impl=MConn{128.199.121.8:46656}
I[05-08|14:06:28.880] Added peer                                   module=p2p peer="Peer{MConn{128.199.121.8:46656} f4c98b895e6cc456814789fcce1b9112d123e0eb out}"
I[05-08|14:06:28.880] Dialing peer                                 module=p2p address=79466a03e9d4b4648a7dd8cead1fa7121ce76ee3@34.235.130.1:46656
I[05-08|14:06:29.393] Successful handshake with peer               module=p2p peer=34.235.130.1:46656 peerNodeInfo="NodeInfo{id: 79466a03e9d4b4648a7dd8cead1fa7121ce76ee3, moniker: macbook-pro-2.intern, network: gaia-5000 [listen 172.31.25.14:46656], version: 0.19.1 ([amino_version=0.9.8 p2p_version=0.5.0 consensus_version=v1/0.2.2 rpc_version=0.7.0/3 tx_index=on rpc_addr=tcp://0.0.0.0:46657])}"
I[05-08|14:06:29.394] Starting Peer                                module=p2p peer=34.235.130.1:46656 impl="Peer{MConn{34.235.130.1:46656} 79466a03e9d4b4648a7dd8cead1fa7121ce76ee3 out}"
I[05-08|14:06:29.394] Starting MConnection                         module=p2p peer=34.235.130.1:46656 impl=MConn{34.235.130.1:46656}
I[05-08|14:06:29.394] Added peer                                   module=p2p peer="Peer{MConn{34.235.130.1:46656} 79466a03e9d4b4648a7dd8cead1fa7121ce76ee3 out}"
I[05-08|14:06:29.394] Dialing peer                                 module=p2p address=1ebc5ca705b3ae1c06a0888ff1287ada82149dc3@138.68.77.24:46656
I[05-08|14:06:29.507] Stopping queryMaj23Routine for peer          module=consensus peer="Peer{MConn{34.205.39.35:46656} eae1dcdfa096571001072417147df0b686fdff48 out}"
I[05-08|14:06:30.064] Stopping queryMaj23Routine for peer          module=consensus peer="Peer{MConn{18.208.82.127:46656} 801bdd7d17e01466feeae9e6d4d46caa1d0c9e0c out}"
I[05-08|14:06:30.206] Timed out                                    module=consensus dur=3s height=92671 round=0 step=RoundStepPropose
I[05-08|14:06:30.208] enterPrevote(92671/0). Current: 92671/0/RoundStepPropose module=consensus
I[05-08|14:06:30.208] enterPrevote: ProposalBlock is nil           module=consensus height=92671 round=0
I[05-08|14:06:30.228] Successful handshake with peer               module=p2p peer=138.68.77.24:46656 peerNodeInfo="NodeInfo{id: 1ebc5ca705b3ae1c06a0888ff1287ada82149dc3, moniker: gaia-5000, network: gaia-5000 [listen 138.68.77.24:46656], version: 0.19.1 ([amino_version=0.9.8 p2p_version=0.5.0 consensus_version=v1/0.2.2 rpc_version=0.7.0/3 tx_index=on rpc_addr=tcp://0.0.0.0:46657])}"
I[05-08|14:06:30.228] Starting Peer                                module=p2p peer=138.68.77.24:46656 impl="Peer{MConn{138.68.77.24:46656} 1ebc5ca705b3ae1c06a0888ff1287ada82149dc3 out}"
I[05-08|14:06:30.228] Starting MConnection                         module=p2p peer=138.68.77.24:46656 impl=MConn{138.68.77.24:46656}
I[05-08|14:06:30.228] Added peer                                   module=p2p peer="Peer{MConn{138.68.77.24:46656} 1ebc5ca705b3ae1c06a0888ff1287ada82149dc3 out}"
I[05-08|14:06:30.228] Dialing peer                                 module=p2p address=eae1dcdfa096571001072417147df0b686fdff48@34.205.39.35:46656
I[05-08|14:06:30.315] Stopping queryMaj23Routine for peer          module=consensus peer="Peer{MConn{207.154.243.140:46656} 45226d5ee2d27c93ec64a674f6bbf6c8f6f598c5 out}"
I[05-08|14:06:30.430] Added to prevote                             module=consensus vote="Vote{2:489222C1FA4C 92671/00/1(Prevote) 000000000000 /EDFD106DF698.../ @ 2018-05-08T14:06:30.242Z}" prevotes="VoteSet{H:92671 R:0 T:1 +2/3:<nil> BA{14:__x___________} map[]}"
I[05-08|14:06:30.481] Added to prevote                             module=consensus vote="Vote{5:A1BDEA32E5B9 92671/00/1(Prevote) 000000000000 /DDFF1ABE147C.../ @ 2018-05-08T14:06:30.208Z}" prevotes="VoteSet{H:92671 R:0 T:1 +2/3:<nil> BA{14:__x__x________} map[]}"
I[05-08|14:06:30.491] Added to prevote                             module=consensus vote="Vote{10:E0EBF116FC88 92671/00/1(Prevote) 000000000000 /71618CCD2CC5.../ @ 2018-05-08T14:06:30.228Z}" prevotes="VoteSet{H:92671 R:0 T:1 +2/3:<nil> BA{14:__x__x____x___} map[]}"
I[05-08|14:06:30.525] Added to prevote                             module=consensus vote="Vote{0:22F03A4B0F6B 92671/00/1(Prevote) 000000000000 /21F8D3F19595.../ @ 2018-05-08T14:06:30.218Z}" prevotes="VoteSet{H:92671 R:0 T:1 +2/3:<nil> BA{14:x_x__x____x___} map[]}"
I[05-08|14:06:30.527] Added to prevote                             module=consensus vote="Vote{9:CF217B72C54F 92671/00/1(Prevote) 000000000000 /E48BFEDEBE58.../ @ 2018-05-08T14:06:30.251Z}" prevotes="VoteSet{H:92671 R:0 T:1 +2/3:<nil> BA{14:x_x__x___xx___} map[]}"
I[05-08|14:06:30.545] Added to prevote                             module=consensus vote="Vote{1:295C0821D6D2 92671/00/1(Prevote) 000000000000 /24B03C82CD9E.../ @ 2018-05-08T14:06:30.249Z}" prevotes="VoteSet{H:92671 R:0 T:1 +2/3:<nil> BA{14:xxx__x___xx___} map[]}"
I[05-08|14:06:30.743] Successful handshake with peer               module=p2p peer=34.205.39.35:46656 peerNodeInfo="NodeInfo{id: eae1dcdfa096571001072417147df0b686fdff48, moniker: Mythos, network: gaia-5000 [listen 10.255.237.165:46656], version: 0.19.1 ([amino_version=0.9.8 p2p_version=0.5.0 consensus_version=v1/0.2.2 rpc_version=0.7.0/3 tx_index=on rpc_addr=tcp://0.0.0.0:46657])}"
I[05-08|14:06:30.743] Starting Peer                                module=p2p peer=34.205.39.35:46656 impl="Peer{MConn{34.205.39.35:46656} eae1dcdfa096571001072417147df0b686fdff48 out}"
I[05-08|14:06:30.743] Starting MConnection                         module=p2p peer=34.205.39.35:46656 impl=MConn{34.205.39.35:46656}
E[05-08|14:06:30.743] Connection failed @ recvRoutine (reading byte) module=p2p peer=34.205.39.35:46656 conn=MConn{34.205.39.35:46656} err=EOF
I[05-08|14:06:30.743] Stopping MConnection                         module=p2p peer=34.205.39.35:46656 impl=MConn{34.205.39.35:46656}
E[05-08|14:06:30.743] Stopping peer for error                      module=p2p peer="Peer{MConn{34.205.39.35:46656} eae1dcdfa096571001072417147df0b686fdff48 out}" err=EOF
I[05-08|14:06:30.743] Stopping Peer                                module=p2p peer=34.205.39.35:46656 impl="Peer{MConn{34.205.39.35:46656} eae1dcdfa096571001072417147df0b686fdff48 out}"
I[05-08|14:06:30.743] Added peer                                   module=p2p peer="Peer{MConn{34.205.39.35:46656} eae1dcdfa096571001072417147df0b686fdff48 out}"
```
**`/dump_consensus_state` output for consensus bugs**
not available 
**Anything else do we need to know**:
@melekes

**Comment by ebuchman** [comment]
Can we see what it crashed with ?

**Comment by kidinamoto01** [comment]
@ebuchman 
The log told me the node was constantly doing handshakes and disconnect as it's suppose to. 
Then the node just stopped and there was no error on info level.  
What I can see is the CPU & mem usage are quite high.

This is not happening after I upgrade my server.

**Comment by melekes** [comment]
Next time please attach the output of `kill -6 <PID>`, where `<PID>` is the tendermint process ID. It will help us understand why node is blocked.

**Comment by xla** [comment]
@kidinamoto01 Can you share the specs of the host that the original issue appeared on?

**Comment by awrelll** [comment]
I also have the same issue. Can't keep a seed node up and running for more than 20hrs.

 ```
I[06-23|08:02:57.312] enterPrevote: ProposalBlock is valid         module=consensus height=368800 round=0
I[06-23|08:03:00.284] Timed out                                    module=consensus dur=3s height=368800 round=0 step=RoundStepPropose
E[06-23|08:03:23.770] Connection failed @ sendRoutine              module=p2p peer=31.187.71.49:46656 conn=MConn{31.187.71.49:46656} err="pong timeout"
I[06-23|08:03:23.771] Stopping MConnection                         module=p2p peer=31.187.71.49:46656 impl=MConn{31.187.71.49:46656}
E[06-23|08:03:23.772] Stopping peer for error                      module=p2p peer="Peer{MConn{31.187.71.49:46656} 121050522471b188dbbb8d659bcaf79983f963b4 out}" err="pong timeout"
I[06-23|08:03:23.772] Stopping Peer                                module=p2p peer=31.187.71.49:46656 impl="Peer{MConn{31.187.71.49:46656} 121050522471b188dbbb8d659bcaf79983f963b4 out}"
I[06-23|08:03:23.846] Stopping gossipVotesRoutine for peer         module=consensus peer="Peer{MConn{31.187.71.49:46656} 121050522471b188dbbb8d659bcaf79983f963b4 out}"
I[06-23|08:03:23.854] Stopping gossipDataRoutine for peer          module=consensus peer="Peer{MConn{31.187.71.49:46656} 121050522471b188dbbb8d659bcaf79983f963b4 out}"
I[06-23|08:03:27.620] Stopping queryMaj23Routine for peer          module=consensus peer="Peer{MConn{31.187.71.49:46656} 121050522471b188dbbb8d659bcaf79983f963b4 out}"
E[06-23|08:04:32.266] Connection failed @ sendRoutine              module=p2p peer=209.97.166.30:46656 conn=MConn{209.97.166.30:46656} err="pong timeout"
I[06-23|08:04:32.267] Stopping MConnection                         module=p2p peer=209.97.166.30:46656 impl=MConn{209.97.166.30:46656}
E[06-23|08:04:32.268] Stopping peer for error                      module=p2p peer="Peer{MConn{209.97.166.30:46656} 8a2408cf7251f96ddc70c4e3c41869e3dc8b5a2a out}" err="pong timeout"
I[06-23|08:04:32.268] Stopping Peer                                module=p2p peer=209.97.166.30:46656 impl="Peer{MConn{209.97.166.30:46656} 8a2408cf7251f96ddc70c4e3c41869e3dc8b5a2a out}"
I[06-23|08:04:32.359] Stopping gossipDataRoutine for peer          module=consensus peer="Peer{MConn{209.97.166.30:46656} 8a2408cf7251f96ddc70c4e3c41869e3dc8b5a2a out}"
I[06-23|08:04:32.360] Stopping gossipVotesRoutine for peer         module=consensus peer="Peer{MConn{209.97.166.30:46656} 8a2408cf7251f96ddc70c4e3c41869e3dc8b5a2a out}"
I[06-23|08:04:33.472] Stopping queryMaj23Routine for peer          module=consensus peer="Peer{MConn{209.97.166.30:46656} 8a2408cf7251f96ddc70c4e3c41869e3dc8b5a2a out}"

```

**Comment by ebuchman** [comment]
This should be fixed in v0.22
